### DISCUSSION TITLE
language

### DISCUSSION THREAD

**Discussion by ming11221122** [BG-Software-LLC/SuperiorSkyblock2#1112] [Approved Feature Requests]
Hello Ome_R could you add a function that can be force players to use a certain language when login?

**Comment by OmerBenGera** [comment]
> Hello Ome_R could you add a function that can be force players to use a certain language when login?

You can change the default language in the config file which new players will have that language set to them

**Reply by ming11221122** [reply]
yes I know about that, but the existing players would not be changed, they need to use </is lang> command to change it,  the default-language only work for new players. :(
